lib/app/modules/home/views/home_view.dart

```diff
@@ -391,6 +393,11 @@ class HomeView extends GetView<HomeController> {
                 ),
               ),
               // LanguageMenu(),
+              Divider(
```

Comment:
Let's remove this divider.

Reply:
Sir can we tweak the divider, Or should I remove it completely?